src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift

```diff
@@ -50,25 +49,19 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 	// MARK: - Protocol ENANavigationControllerWithFooterChild
 
 	func navigationController(_ navigationController: ENANavigationControllerWithFooter, didTapPrimaryButton button: UIButton) {
-		callHotline()
-	}
-
-	func navigationController(_ navigationController: ENANavigationControllerWithFooter, didTapSecondaryButton button: UIButton) {
-		self.onSecondaryButtonTap()
+		self.onPrimaryButtonTap()
 	}
 
 	// MARK: - Private
 	
-	private let onSecondaryButtonTap: () -> Void
+	private let onPrimaryButtonTap: () -> Void
 
 	private lazy var navigationFooterItem: ENANavigationFooterItem = {
 		let item = ENANavigationFooterItem()
-		item.primaryButtonTitle = AppStrings.ExposureSubmissionHotline.callButtonTitle
+		item.primaryButtonTitle = AppStrings.ExposureSubmissionHotline.tanInputButtonTitle
 		item.isPrimaryButtonEnabled = true
```

Comment:
do we need here additionally `item.isPrimaryButtonHidden = false`?

Reply:
@KaiTeuber Do you think we need this?